workflows/rnaseq/main.nf

```diff
@@ -787,6 +787,385 @@ workflow RNASEQ {
     versions       = ch_versions       // channel: [ path(versions.yml) ]
 }
 
+output {
+    directory params.outdir, mode: params.publish_dir_mode
+
+    //
+    // STAR Salmon alignment options
+    //
+    "${params.aligner}" {
```

Comment:
I think it should be avoided the use of expressions, this makes it much more inscrutable

Reply:
only strings and dynamic strings are allowed. not sure else you would model it, this is how rnaseq defines the directory structure

Reply:
we can have:
```
if params.aligner = 'salmon'
"salmon/"
```

and so on

Reply:
Isn't that just a dynamic string with extra steps? 😆